cluster/gce/util.sh

```diff
@@ -167,13 +188,26 @@ function upload-server-tars() {
 
   local -r staging_path="${staging_bucket}/devel"
 
+  local server_hash=$(sha1sum-file ${SERVER_BINARY_TAR})
```

Comment:
See my comment above about `local var=$(...)`, it masks the error of sha1sum-file but ideally you want an error there to break your script. And, if you don't, you should make that explicit with `|| :`.

Also, the arguments should be double quoted, they're supposed to expand to a single word.

```shell
  local server_hash salt_hash
  server_hash=$(sha1sum-file "${SERVER_BINARY_TAR}")
  salt_hash=$(sha1sum-file "${SALT_TAR}")
```